manifests/base/runtimes/pretraining/mpi_distributed.yaml

```diff
@@ -11,35 +11,47 @@ spec:
     mpi:
       numProcPerNode: 1
       mpiImplementation: OpenMPI
-      sshAuthMountPath: /root/.ssh
+      sshAuthMountPath: /home/mpiuser/.ssh
   template:
     spec:
+      network:
+        publishNotReadyAddresses: false
+      successPolicy:
+        operator: All
+        targetReplicatedJobs:
+          - launcher
       replicatedJobs:
         - name: launcher
           template:
             spec:
               template:
                 spec:
-                  # TODO (andreyvelich): Change the command with mpirun.
                   containers:
                     - name: launcher
-                      image: busybox
+                      image: mpioperator/mpi-pi:openmpi
```

Comment:
Are these image published from kubeflow?

Reply:
Yes, here is the Dockerfile: https://github.com/kubeflow/mpi-operator/tree/master/examples/v2beta1/pi

Reply:
Does kubeflow have an official repository for this? Otherwise this hits dockerhub and you may get rate limited if you brought this in the CI.

Reply:
I will open an issue for that in mpi-operator repository.

Reply:
This one: https://github.com/kubeflow/mpi-operator/issues/684